__tests__/db.test.js

```diff
@@ -41,4 +41,39 @@ describe('/api', () => {
             })
         })
     });
+});
+
+describe('/api/reviews/:review_id', () => {
+    test('GET - status: 200 - responds with correct review object', () => {
+        return request(app)
+        .get("/api/review/3")
+        .expect(200)
+        .then((result) => {
+            expect(result.body.review).toEqual({ review_id: 3, title: "Ultimate Werewolf", category: "social deduction", designer: "Akihisa Okui", owner: "bainesface", review_body: "We couldn't find the werewolf!", review_img_url: "https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?w=700&h=700", created_at: "2021-01-18T10:01:41.251Z", votes: 5});
+        })
+    });
+    test('GET - status: 400 - gives correct error when given an invalid review_id number', () => {
```

Comment:
Line 55 - This test should be a 404. A 404 is used when something _could_ exist but doesn't yet, for example the ID of 666 could one day be present in the DB, so it's not considered a 400 bad request. Where as review/woof is a bad request because it would never exist as an endpoint without the dev adding it